cogs/StaffAppsBackoffice.py

```diff
@@ -0,0 +1,966 @@
+        interaction: discord.Interaction,
+    ) -> None:
+        self.cur_state = not self.cur_state
+        await interaction.response.edit_message(
+            embed=self.embeds[self.cur_state].build(),
+            view=self,
+        )
+
+
+SEE_GUILD = getenv("ALLOW_DUMP_DATABASE_GUILD")
+SEE_ROLE = getenv("ALLOW_SEE_APPS_ROLE")
+SEE_PERMISSIONS = (lambda x: x) if SEE_ROLE is None else commands.has_role(SEE_ROLE)
+SUBCOMMANDS = ["all", "spam", "accepted", "denied"]
+
+
+class StaffAppsBackoffice(commands.Cog):
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.db = sqlite3.connect("util/database.sqlite")
+        self.cursor = self.db.cursor()
+
+    @SEE_PERMISSIONS
+    @commands.slash_command(
+        name="see-apps",
+        description="Command to check and work with applications.",
+        guild_ids=None if SEE_GUILD is None else [SEE_GUILD],
+        guild_only=SEE_GUILD is not None,
+    )
+    @option(
+        "subcommand",
+        str,
+        description="The subcommand to use. (may be empty)",
+        required=False,
+        choices=SUBCOMMANDS,
+    )
+    @option(
+        "specific_id",
+        int,
+        description="The specific application ID to check. IGNORES SUBCOMMAND",
+        required=False,
+    )
+    async def see_apps(
+        self,
+        ctx: commands.Context,
+        subcommand: str,
+        specific_id: int,
+    ) -> None:
+        username = f"{ctx.author.name}#{ctx.author.discriminator}"
+        Log(f"see-apps command used by {username}:({ctx.author.id})")
+        if specific_id:
+            data = self.cursor.execute(
+                "select a.appId, a.uid, a.discordName, a.firstName, a.timezone, a.hoursAvailableWk, a.staffReason, a.contributeReason, a.submissionTime, s.description from application a join status s on a.status = s.statusId where appId = ?;",
+                (specific_id,),
+            ).fetchone()
+            if not data:
+                await ctx.send("Application not found.", ephemeral=True)
+                return
+            like_list = [
+                x[0]
+                for x in self.cursor.execute(
+                    "select likes from like where appId = ?;",
+                    (specific_id,),
+                ).fetchall()
+            ]
+            likes = like_list.count(1)
+            dislikes = like_list.count(0)
+            title = "Staff application"
+            description = f"Application ID: **{data[0]}**\nApplicant name: **{data[2]}**\nApplicant ID: **{data[1]}**\n"
+            fields = [
+                ["Status", data[9], False],
+                ["first name", data[3], False],
+                ["Time zone", data[4], False],
+                ["Hours available per week", data[5], False],
+                ["Why staff?", data[6][:1024], False],
+                ["Contribute reason", data[7][:1024], False],
+                ["Submission time", datetime.fromtimestamp(int(data[8])), False],
```

Comment:
datetime.fromtimestamp() takes an EPOCH timestamp as argument, EPOCH timestamps are the time in seconds elapsed since 00:00:00 UTC on 1 January 1970. This object is already timezone-aware at the time of creation.